syntax = "proto3";

message PersonsMessage {
    int32 id = 1;
    string first_name = 2;
    string last_name = 3;
    string company_name = 4;
}



service CallService {
    rpc Create(PersonsMessage) returns (PersonsMessage);
}